package/Linux/gateway/template/rules

```diff
@@ -12,5 +12,6 @@ override_dh_install:
 	dh_install
 	mkdir -p $$(pwd)/debian/devolutions-gateway/usr/share/devolutions-gateway/webapp
 	cp -r {{ dgateway_webclient }} $$(pwd)/debian/devolutions-gateway/usr/share/devolutions-gateway/webapp
+	cp -r {{ root_path }}/webapp/player $$(pwd)/debian/devolutions-gateway/usr/share/devolutions-gateway/webapp
```

Comment:
I think you are missing /player at the end of this line?

Reply:
Screenshot is the structure of the .deb produced in CI/Packaging

Reply:
Oh yes sorry about that you are correct.